sections/related_works.tex

```diff
@@ -37,11 +44,8 @@ \section{Related Work}\label{sec:related_work}
 WF-LDA extends \gls{lda} by using word features to make a prior for the topics.
 LF-LDA takes the approach of replacing \gls{lda}'s topic-word Dirichlet multinomial component with a two-component mixture of a topic-word Dirichlet multinomial component and a latent feature component.
 
-From these works, we mainly rely on the concepts from \citet{author_topic_2012} for how we use metadata in our models, though with slight changes in the models for handling the characteristics of our data.
-There is one main reason why we use the concepts from the author-topic model instead of the newer MetaLDA model~\cite{MetaLDA2017}.
-In MetaLDA, each document has a specific Dirichlet prior for its topic distribution, which is computed from the metadata of the document.
+From these works, we mainly use the concepts from \citet{author_topic_2012} for how we use metadata in our models, though with slight changes in the models for handling the characteristics of our data.
+As mentioned earlier, one of the main reason why we use the concepts from the Author-Topic model instead of the newer MetaLDA model~\cite{MetaLDA2017} is that in MetaLDA, each document has a specific Dirichlet prior for its topic distribution, which is computed from the metadata of the document.
```

Comment:
```suggestion
From these works, we mainly use the concepts from \citet{author_topic_2012} for how we use metadata in our models, though with slight changes in the models for handling the characteristics of our data.
As mentioned earlier, one of the main reasons why we use the concepts from the Author-Topic model instead of the newer MetaLDA model~\cite{MetaLDA2017} is that in MetaLDA, each document has a specific Dirichlet prior for its topic distribution, which is computed from the metadata of the document.
```